content-scripts/inject/run-userscript.js

```diff
@@ -1,7 +1,7 @@
 import Addon from "../../addon-api/content-script/Addon.js";
 
-export default async function runAddonUserscripts({ addonId, scripts, traps }) {
-  const addonObj = new Addon({ id: addonId, traps });
+export default async function runAddonUserscripts({ addonId, scripts, traps, enabledLate }) {
```

Comment:
Forgot a default value for `enabledLate`? Should be `false`.
Also, I think we can remove the `traps` parameter which is now unused.